dev/src/util/PomUtilities.ts

```diff
@@ -41,12 +41,7 @@ export default class PomUtilities {
                 };
                 var generatedFile = await Utils.getUniqueFileName(filePath, "pom-generated", ".xml");
                 // Code gen just occurred so the pom.xml now is from the generator.
-                await fs.copyFile(filePath + "/pom.xml", filePath + "/" + generatedFile, async (e: any) => {
-                    if (e) {
-                        Log.e(e);
-                        reject(false);  // Can't merge
-                    }
-                });
+                await fs.renameSync(filePath + "/pom.xml", filePath + "/" + generatedFile);
```

Comment:
@sghung , I noticed some async issues and the fix is limited to PomUtilities.ts.  I've included it here.  I'm simply using renameSync instead of copyFile and it does the trick.  I've tested it on Mac and Windows.  The other changes below are simply adding `await` to the calls, so these are safe.